src/components/Wrapper/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import cx from 'classnames'
+import './styles.css'
+
+const Wrapper = props => {
+  const { className, ...rest } = props
+  return <div className={cx('Wrapper', className)} {...rest} />
+}
+
```

Comment:
Should this thing be moved to cozy-ui ? Wrapper is too generic a name for me here.

Reply:
Actually it's like the existing `NarrowContent`, but a little less narrow, since it's max 768px wide, where NarrowContent is max 512px wide. Maybe a variant of NarrowContent?